Tests/Translation/Extractor/TwigExtractorTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
```

Comment:
This repository is supposed to be MIT https://github.com/PrestaShop/TranslationToolsBundle/blob/master/composer.json#L5